forge-gui/res/cardsfolder/upcoming/smash_to_dust.txt

```diff
@@ -0,0 +1,8 @@
+Name:Smash to Dust
+ManaCost:1 R
+Types:Sorcery
+A:SP$ Charm | Choices$ DBDestroyArtifact,DBDestroyDefender,DBDamage
+SVar:DBDestroyArtifact:DB$ Destroy | ValidTgts$ Artifact | Tgtprompt$ Select target artifact | SpellDescription$ Destroy target artifact
+SVar:DBDestroyDefender:DB$ Destroy | ValidTgts$ Creature.withDefender | TgtPrompt$ Select target creature with defender. | SpellDescription$ Destroy target creature with defender.
+SVar:DBDamage:DB$ DamageAll | NumDmg$ 1 | ValidCards$ Creature.OppCtrl | ValidDescription$ Each creature your opponents control. | SpellDescription$ CARDNAME deals 1 damage to each creature your opponents control.
```

Comment:
`ValidDescription$ Each` > `ValidDescription$ each` - this param is for building the stack description and always puts this at the end of a sentence so we don't want a cap here